articles/quickstart/spa/vuejs/_includes/_centralized_login.md

```diff
@@ -145,9 +145,8 @@ export const useAuth0 = ({
     async created() {
       // Create a new instance of the SDK client using members of the given options object
       this.auth0Client = await createAuth0Client({
-        domain: options.domain,
+        ...options,
```

Comment:
Just out of curiosity, why did we move to spreading the options? Even tho it is not problematic, we are passing an object to `createAuth0Client` that has both `clientId` and `client_id` as a property.

Reply:
Mainly because without this you can't pass any additional options to the SDK, like the refresh token config or storage mechanism.

It is a little weird at the moment but this not the whole story - this will be cleaned up when we extract the wrapper out into its own SDK.